bundles/org.openhab.binding.wlanthermo/src/main/java/org/openhab/binding/wlanthermo/internal/api/esp32/WlanThermoEsp32CommandHandler.java

```diff
@@ -0,0 +1,298 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.wlanthermo.internal.api.esp32;
+
+import static org.openhab.binding.wlanthermo.internal.WlanThermoBindingConstants.*;
+
+import java.awt.Color;
+import java.math.BigInteger;
+import java.util.List;
+
+import javax.measure.Unit;
+import javax.measure.quantity.Temperature;
+
+import org.openhab.binding.wlanthermo.internal.api.esp32.dto.data.Channel;
+import org.openhab.binding.wlanthermo.internal.api.esp32.dto.data.Data;
+import org.openhab.binding.wlanthermo.internal.api.esp32.dto.data.Pm;
+import org.openhab.binding.wlanthermo.internal.api.esp32.dto.data.System;
+import org.openhab.binding.wlanthermo.internal.api.esp32.dto.settings.Settings;
+import org.openhab.core.library.types.*;
+import org.openhab.core.library.unit.ImperialUnits;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.openhab.core.types.UnDefType;
+
+/**
+ * The {@link WlanThermoEsp32CommandHandler} is responsible for mapping the Commands to the respective data fields
+ * of the API.
+ *
+ * @author Christian Schlipp - Initial contribution
+ */
+public class WlanThermoEsp32CommandHandler {
+
+    public State getState(ChannelUID channelUID, Data data, Settings settings) {
```

Comment:
This class is stateless, so I see no reason you couldn't make all of it's method static.
```suggestion
    public static State getState(ChannelUID channelUID, Data data, Settings settings) {
```